### PR TITLE
--quick option for local docker build

### PR DESCRIPTION
**Pull request by coltmcnealy-lh**
Adds a --quick option to local-dev/build.sh which uses local gradle cache to build.

### CODE REVIEW

File: local-dev/build.sh [hunk 1]
```diff
@@ -9,10 +9,28 @@ CONTEXT_DIR=$(cd "$SCRIPT_DIR/.." && pwd)
 
 cd "$CONTEXT_DIR"
 
-echo "Building server"
-docker build --file ${LH_SERVER_WORK_DIR}/Dockerfile \
-    --tag littlehorse/littlehorse-server:latest ${CONTEXT_DIR}
+CONTAINER=lh-quickbuilder
```
Comment (sauljabin) [inline local-dev/build.sh]: What is this for?

File: local-dev/util/Dockerfile.server-quick [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM amazoncorretto:17
```
Comment (sauljabin) [inline local-dev/util/Dockerfile.server-quick:1]: we can encapsulate this inside the same script:

example:

 docker build -f ${SCRIPT_DIR}/util/Dockerfile.server-quick \
        --tag littlehorse/littlehorse-server:lates -<<EOF
WORKDIR /lh

COPY ./docker/server/docker-entrypoint.sh /lh
COPY ./docker/server/log4j2.properties /lh
COPY ./server/build/libs/server-*-all.jar /lh/server.jar
ENTRYPOINT ["/lh/docker-entrypoint.sh"]
CMD ["server"]
EOF

Comment (sauljabin) [reply local-dev/util/Dockerfile.server-quick:1]: it's just a suggestion hehe

**Review by sauljabin — APPROVED** [review]

File: local-dev/util/Dockerfile.server-quick (hunk 2 above)
Comment (coltmcnealy-lh) [reply local-dev/util/Dockerfile.server-quick:1]: Roger Roger. Making that change then merging.

Comment (coltmcnealy-lh) [reply local-dev/util/Dockerfile.server-quick:1]: I got an error after playing around with it for a few minutes, and I don't see the option here: https://docs.docker.com/engine/reference/commandline/build/

I'm going to merge as-is, and we can improve it later.

**Pull request merged**